packages/core/scripts/release-prod.sh

```diff
@@ -46,7 +46,7 @@ then
 fi
 
 message "Installing packages" &&
-lerna bootstrap --ci && lerna run build --scope=deriv-shared --scope=deriv-components &&
+lerna bootstrap --ci && lerna run build:travis &&
```

Comment:
this will also install trader, core, bot, and p2p?

Reply:
It will only install p2p, shared, translations, and components. Please refer to `docs/README.md` file for details.